docs/docs/use-cases/hpa.md

```diff
@@ -91,7 +89,7 @@ Status:
 
 Here we can see that the value of the `cpu-throttling` metric is `1.63`
 
-### Set up the HorizontalPodAutoscaler
+## Set up the HorizontalPodAutoscaler
 
 Now that we are able to retrieve the value of our metric, and have it stored in
 our cluster in the status of our `KeptnMetric` custom resource, we can configure
```

Comment:
```suggestion
our cluster in the status of our `KeptnMetric` resource, we can configure
```
A few months ago, I think we agreed that we would just call these resources in general prose like this.

Reply:
done